CHANGES.md

```diff
@@ -1,3 +1,33 @@
+1.7.2 / 2019-09-XX
+==================
+
+Bug fixes:
+
+* DriverInterface#getValue() can return null
```

Comment:
Please remove this line.

---

The method returned NULL before, but that wasn't mentioned in the DocBlock. DocBlock change isn't something, that goes into change log ;)